website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md

```diff
@@ -55,21 +55,21 @@ When you have configured your host to use Podman rather then Docker, consider ve
      <TabItem value="win" label="Windows">
 
    ```shell-session
-   $ CONTAINER_HOST=npipe:////./pipe/docker_engine podman ps
+   $ CONTAINER_HOST=npipe:////./pipe/docker_engine; podman ps
    ```
 
      </TabItem>
      <TabItem value="mac" label="macOS">
 
    ```shell-session
-   $ CONTAINER_HOST=/var/run/docker.sock podman ps
+   $ CONTAINER_HOST=/var/run/docker.sock; podman ps
    ```
 
      </TabItem>
      <TabItem value="linux" label="Linux">
 
    ```shell-session
-   $ CONTAINER_HOST=/var/run/docker.sock podman ps
+   $ CONTAINER_HOST=/var/run/docker.sock; podman ps
```

Comment:
Not required on Linux

Reply:
It actually breaks the example, since it is no longer setting the CONTAINER_HOST environment variable.

But it was broken on many podman versions anyway, since it doesn't use podman-remote or --remote.